docs/src/pages/[platform]/components/tabs/examples/BasicTabs.tsx

```diff
@@ -4,5 +4,12 @@ export const BasicTabs = () => (
   <Tabs>
     <TabItem title="Tab 1">Tab 1 Content</TabItem>
     <TabItem title="Tab 2">Tab 2 Content</TabItem>
+    <TabItem title="Tab 3">Tab 3 Content</TabItem>
```

Comment:
?

Reply:
Not sure how it's got sneaked in, removed